google-http-client-apache-legacy/src/main/java/com/google/api/client/http/apache/ApacheHttpTransport.java

```diff
@@ -61,20 +61,16 @@
 /**
  * Thread-safe HTTP transport based on the Apache HTTP Client library.
  *
- * <p>
- * Implementation is thread-safe, as long as any parameter modification to the
- * {@link #getHttpClient() Apache HTTP Client} is only done at initialization time. For maximum
- * efficiency, applications should use a single globally-shared instance of the HTTP transport.
- * </p>
+ * <p>Implementation is thread-safe, as long as any parameter modification to the {@link
```

Comment:
Please have the `{@link` as the same line as the link itself.

Reply:
Commandline does not work as expected `mvn com.coveo:fmt-maven-plugin:format` so resolved through manual.